docs/features/vendored-dependencies.md

```diff
@@ -125,6 +125,76 @@ Due to this caching setup, it is normal for the first analysis to take some time
 
 If `version` is not specified, FOSSA computes a version based on the contents specified by `path`; this means that if the contents have not changed then the results are reused.
 
-In the event caching is causing problems, FOSSA can be made to rescan this kind of dependency: 
+In the event caching is causing problems, FOSSA can be made to rescan this kind of dependency:
 - Run `fossa analyze` with the `--force-vendored-dependency-rescans` flag, or
 - Set `vendoredDependencies.forceRescans` to `true` in `.fossa.yml` at the root of the project.
+
+## Path Filtering
+
+> Note: This section does not apply to archive uploads. Path filtering is only available when doing a CLI License Scan. See [here](#how-vendored-dependencies-are-scanned) for more info on the difference between these two methods.
+
+Path filtering can be used to omit some files or directories from license scanning. Path filtering is set up in the `.fossa.yml` file. Here is an example:
```

Comment:
[Optional] I realize this will very large uplift, so feel free to ignore this completely.

It would be nice to include filtering in the scan summary (whether any files were filtered, or whether filer was applied). 

I'm thinking of a scenario, where the user applies some filtering - but applies it incorrectly (glob fmt etc.) - how can they confirm that the filtering they are applying is correct for their behaviour? If files excluded by globs are included in the raw themis output - maybe we can include it in the doc, for confirmation purposes.

Reply:
Dang it - I realize that triaging bit is included in the PR description 😅

Reply:
I had put that in the ACs, but had not actually added the "how to do it" to the docs. Thank you for the reminder!

Reply:
I've added that to the docs now